diff --git a/node_modules/next-themes/package.json b/node_modules/next-themes/package.json
index 5a55392..036d548 100644
--- a/node_modules/next-themes/package.json
+++ b/node_modules/next-themes/package.json
@@ -62,6 +62,6 @@
     "react-dom": "^16.8 || ^17 || ^18"
   },
   "peerDependencies": {
-    "react": "^16.8 || ^17 || ^18",
+    "react": "^16.8 || ^17 || ^18 || ^19",
     "react-dom": "^16.8 || ^17 || ^18"
   },
   "resolutions": {
diff --git a/node_modules/next-themes/dist/index.js b/node_modules/next-themes/dist/index.js
index 9b7c89f..1d91834 100644
--- a/node_modules/next-themes/dist/index.js
+++ b/node_modules/next-themes/dist/index.js
@@ -10,7 +10,7 @@
     "use client"
     var import_react = require("react");
     var import_react_dom = require("react-dom");
-    var __defProp = Object.defineProperty;
+    var __defProp = Object.defineProperty; 
     var __getOwnPropDesc = Object.getOwnPropertyDescriptor;
     var __getOwnPropNames = Object.getOwnPropertyNames;
     var __hasOwnProp = Object.prototype.hasOwnProperty;
diff --git a/node_modules/next-themes/dist/types.d.ts b/node_modules/next-themes/dist/types.d.ts
index c733989..d143c7b 100644
--- a/node_modules/next-themes/dist/types.d.ts
+++ b/node_modules/next-themes/dist/types.d.ts
@@ -1,6 +1,6 @@
-import type { ReactNode, Dispatch, SetStateAction } from 'react';
+import type { ReactNode, Dispatch, SetStateAction, ReactElement } from 'react';
 export interface ThemeProviderProps {
-    children?: ReactNode;
+    children?: ReactNode | ReactElement;
     /** List of all available theme names */
     themes?: string[];
     /** Forced theme name for the current page */
